extensions/wikia/Chat2/tests/ChatWidgetTest.php

```diff
@@ -95,6 +95,7 @@ public function testGetTemplateVars( $fromParserTag, $wgEnableWallExt, $isLogged
 		// static method mocks
 		$this->mockStaticMethod( 'ChatWidget', 'getUsersInfo', $chatUsersInfo );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "ChatWidget" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)